testing/adios2/engine/bp/operations/TestBPWriteReadZfpCuda.cpp

```diff
@@ -4,8 +4,11 @@
  */
 
 #include "CudaRoutines.h"
+
 #include <adios2.h>
 
+#include <cuda_runtime.h>
```

Comment:
I would maybe leave this in the `CudaRoutines.h` file because all kernels will be here so most likely all tests using Cuda will include this file.

Reply:
Good idea